htmx-spring-boot/src/main/java/io/github/wimdeblauwe/htmx/spring/boot/mvc/HtmxTrigger.java

```diff
@@ -0,0 +1,46 @@
+package io.github.wimdeblauwe.htmx.spring.boot.mvc;
+
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Represents a value of HX-Trigger, HX-Trigger-After-Settle or HX-Trigger-After-Swap.
+ *
+ * @see <a href="https://htmx.org/headers/hx-trigger/">HX-Trigger Response Headers</a>
+ */
+public class HtmxTrigger {
```

Comment:
This could be a record I think since it is immuable?

Reply:
I am not sure if we should really use a record. These are not very flexible with regard to any future adjustments.